### DISCUSSION TITLE
MP4 format

### DISCUSSION THREAD

**Discussion by tonnyinocencio** [shaked6540/YoutubePlaylistDownloader#149] [Q&A] [answered]
First of all, thanks for building this fantastic tool. I would like to know how to download in mp4 format

**Comment by shaked6540** [comment]
There is no way to download in mp4 format currently, because mp4 does not support subtitles. I might add that in the future but currently I don't have time, sorry. If you know how to code you can make a pull request for it

**Comment by shaked6540** [comment, marked as answer]
Mp4 support was added in version 1.9.19

**Reply by tonnyinocencio** [reply]
excellent work, thank you very much.
